remote-api/src/main/java/org/alfresco/rest/api/impl/AspectsImpl.java

```diff
@@ -151,9 +148,8 @@ public Aspect convertToAspect(AspectDefinition aspectDefinition, List<String> in
         }
         catch (Exception ex)
         {
-            logger.warn("Failed to parse Aspect " + aspectDefinition.getName().toPrefixString());
+            throw new AlfrescoRuntimeException("Failed to parse Aspect" + aspectDefinition.getName());
```

Comment:
Thanks for changing this but looks like you've lost the space in the error message.